setup.py

```diff
@@ -19,4 +26,16 @@
     data_files=[
         ('share/metainfo', ['com.github.whipper_team.Whipper.metainfo.xml']),
     ],
+    classifiers=[
+        'Development Status :: 4 - Beta',
```

Comment:
Should be changed to `'Development Status :: 5 - Production/Stable'` for 1.0.0, but calling it "beta" until then seems fine, no?

Reply:
Yes